ui/packages/catalog-ui-search/src/main/webapp/component/result-selector/result-selector.view.js

```diff
@@ -100,7 +100,11 @@ define([
     },
     initialize: function(options) {
       if (!this.model.get('result')) {
-        this.model.startSearch()
+        if (options.tieredSearchIds !== undefined) {
```

Comment:
✏️ This can be simplified to...

```
if (options.tieredSearchIds) {
    . . .
}
```

This will evaluate to false if `options.tieredSearchIds` is `undefined`, `null`, `NaN`, `0`, `""`, `false`